Mathlib/Logic/Equiv/Basic.lean

```diff
@@ -832,6 +832,15 @@ def sumPiEquivProdPi {ι ι'} (π : ι ⊕ ι' → Type*) :
   left_inv f := by ext (i | i) <;> rfl
   right_inv _ := Prod.ext rfl rfl
 
+/-- The space of functions from `PUnit` into a family of types
+is isomorphic to the value of this function at `()`.
+-/
+def pUnitPiEquiv (f : PUnit → Type*) : ((t : PUnit) → (f t)) ≃ f () where
```

Comment:
```suggestion
def pUnitPiEquiv (f : PUnit → Type*) : (∀ t : PUnit, f t) ≃ f () where
```
I think this is more readable for a mathematician eye (I would even argue for Pi-notation but most of this file already uses forall).
Same remark for all flavors of `pUnitPiEquiv`

Reply:
Am I going crazy? `(Π t : PUnit, f t)` gives me an `expected token` error on the `\Pi` so that's probably why this file isn't using it! I agree that \Pi is best, it looks like a product (and indeed is a product).

Reply:
You're not going crazy: [PiNotation.piNotation](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Util/Delaborators.html#PiNotation.piNotation) is simply not imported yet!